docs/reference/security/authorization/field-and-document-access-control.asciidoc

```diff
@@ -59,3 +59,26 @@ documents by index instead.
 
 include::role-templates.asciidoc[]
 include::set-security-user.asciidoc[]
+
+
+[[ccx-apikeys-dls-fls]]
+==== Field and document level security with Cross-cluster API keys
+
+<<security-api-create-cross-cluster-api-key, Cross-Cluster API keys>> can be used to authenticate
+requests to a remote cluster.`access` provides access for Cross Cluster Search.
```

Comment:
I think we want `search` instead of `access` here and throughout the paragraphs below.